ReverseLinkedList.java

```diff
@@ -0,0 +1,22 @@
+public class ReverseLinkedList {
+    public ListNode reverseList(ListNode head) {
+           if(head==null || head.next==null) return head;
+            ListNode prev=null;
+            ListNode curr=head;
+            while(curr != null){
```

Comment:
Good job on the iterative approach. Can you also try recursive approach for reversing linkedlist?